pylint/checkers/base.py

```diff
@@ -1987,6 +1987,10 @@ def visit_assignname(self, node):
                     self._check_name("variable", node.name, node)
         elif isinstance(frame, astroid.ClassDef):
             if not list(frame.local_attr_ancestors(node.name)):
+                for ancestor in frame.ancestors():
+                    if ancestor.name == "Enum" and ancestor.root().name == "enum":
+                        self._check_name("const", node.name, node)
+                        return
```

Comment:
I would prefer a simple break statement here instead of return. What do you think about it?

Reply:
I think so too. I fix it.

Reply:
Thanks @Hee-San